medusa/network_timezones.py

```diff
@@ -72,8 +72,11 @@ def update_network_dict():
         return
 
     cache_db_con = db.DBConnection('cache.db')
+    sql_result = cache_db_con.select('SELECT network_name, timezone FROM network_timezones;')
 
-    network_list = dict(cache_db_con.select('SELECT * FROM network_timezones;'))
+    network_list = {}
+    for row in sql_result:
+        network_list[row['network_name']] = row['timezone']
```

Comment:
I would do this instead (instead of lines 77-79):
```py
    network_list = {
        row['network_name']: row['timezone']
        for row in sql_result
    }
```

Reply:
changed